lib/shoryuken/manager.rb

```diff
@@ -122,16 +132,15 @@ def pause_queue!(queue)
       after(Shoryuken.options[:delay].to_f) { async.restart_queue!(queue) }
     end
 
-
     def dispatch
       return if stopped?
 
       logger.debug { "Ready: #{@ready.size}, Busy: #{@busy.size}, Active Queues: #{unparse_queues(@queues)}" }
 
       if @ready.empty?
-        logger.debug { 'Pausing fetcher, because all processors are busy' }
+        logger.debug { 'Pausing fetcher for #{@fetcher_pause_interval} seconds, because all processors are busy' }
```

Comment:
Single-quoted strings are not interpolated, so the log will contain the literal text:

```
Pausing fetcher for #{@fetcher_pause_interval} seconds, because all processors are busy
```

You probably want to use double quotes here.